src/function/table/version/pragma_version.cpp

```diff
@@ -57,6 +57,10 @@ const char *DuckDB::LibraryVersion() {
 	return DUCKDB_VERSION;
 }
 
+const char *DuckDB::Distribution() {
+	return DUCKDB_DISTRIBUTION;
+}
+
```

Comment:
On a second though, should this be part of the configuration options?

Reason is that there are complex situations like:
* DuckDB paired with extension that also packages it's own DuckDB statically: Distribution() is always the same independent of the call side? (say from within extension code or from main code?)
* Local DuckDB paired with remote DuckDB, does each gets it's own Distribution or there is only one (dictated by configuration?

Reply:
That's a good question... I guess there is a sliding scale between a compile-time distribution and a runtime custom user agent. Since `custom_user_agent` is additive, any call-site layer can build on it already as of #9603, so we don't have to make compile-time distribution configurable, too. It can be though, for cases where the term "distribution" better expresses reality than "custom user agent", like in client/server scenario.

@bleskes opinions?

Reply:
I would think it's easier to reason with the system if the same results are obtained even if executed in different part of the code, like with some extensions (packaging a static duckdb) loaded, the result of queries should be independent of which duckdb is actually executing a given query, so it seems sane to have a central point (say configuration) dictate like "first to come will get to choose the distribution", but I might be off on this idea.

Reply:
I agree that distribution should always be reproducible. I am not sure what the implication on making it a configuration option is though -- a pure compile-time constant guarantees this kind of reproducibility but if I make it a configuration option, won't it become mutable?